packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -98,6 +104,25 @@ async fn create_data_type<P: GraphPool>(
     Ok(Json(schema))
 }
 
+#[utoipa::path(
+    get,
+    path = "/data-types",
+    tag = "DataType",
+    responses(
+        (status = 200, content_type = "application/json", description = "Data type found", body = [DataType]),
+        (status = 422, content_type = "text/plain", description = "Provided URI is invalid"),
+
+        (status = 500, description = "Store error occurred"),
+    )
+)]
+async fn get_data_type_unfiltered<P: GraphPool>(
```

Comment:
I do not think this name is intuitive at all :D 

Can we call this something like `get_all_latest_version_data_types` or something like that?